app/views/comments/_single_comment.html.erb

```diff
@@ -59,8 +59,10 @@
         <% end %>
       <% end %><!-- end caching -->
 
+      <% if logged_in_as_admin? %>
+        <p class="comment ip"><%= ts("IP Address: %{ip_address}", ip_address: single_comment.ip_address.blank? ? "No address recorded" : single_comment.ip_address) %></p>
```

Comment:
These should just be `<p class="ip">` -- the `.comment` selector refers to the class on the surrounding `li` -- but that shouldn't be preventing the red from being applied. What does your non-working code look like? Is the right alignment not being applied either?